youtube_dl_gui/locale/ru_RU/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -0,0 +1,443 @@
+# Youtube-dlG localization file.
+# FIRST AUTHOR: Sotiris Papadopoulos <ytubedlg@gmail.com>, 2015.
+# Nikita «Arttse» Bystrov <arttsesoft@gmail.com>, 2015.
+msgid ""
+msgstr ""
+"Project-Id-Version: youtube-dlg 0.3.8\n"
+"POT-Creation-Date: 2015-07-02 15:31+EEST\n"
+"PO-Revision-Date: 2015-11-22 10:38+0300\n"
+"Last-Translator: Nikita «Arttse» Bystrov <arttsesoft@gmail.com>\n"
+"Language-Team: ru\n"
+"Language: ru\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: pygettext.py 1.5\n"
+
+#: mainframe.py:82
+msgid "URLs"
+msgstr "Ссылки (каждая на новой строке)"
+
+#: mainframe.py:83
+msgid "Download"
+msgstr "Скачать"
+
+#: mainframe.py:84
+msgid "Update"
+msgstr "Обновить приложение"
```

Comment:
I wrote it because it is understandably to the user. Nobody will understand what specifically updated. The phrase `Обновить приложение` says that it is necessary to update the program. If you think that you need to change the phrase, I change it to `Обновить`.